### PR TITLE
Added support for node v14

### PR DESCRIPTION
**Pull request by nannanli**
Signed-off-by: nannanli <nannanli@cn.ibm.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [x] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.
Updated the doc to state that Node.js v14 is supported now. 

:heart:Thank you!

FYI - @nkocsis @Joe-Winchester @stevenhorsman

### CODE REVIEW

File: docs/user-guide/systemrequirements.md [hunk 1]
```diff
@@ -123,10 +127,10 @@ The Zowe Desktop supports Google Chrome, Mozilla Firefox, Apple Safari and Micro
 For Firefox, both the regular and Extended Support Release (ESR) versions are supported under this rule.
 
 Currently, the following browsers are supported:
-    -   Google Chrome V77 or later
+    -   Google Chrome V77, V79 or later
     -   Mozilla Firefox V68 or later
-    -   Safari V12.1 or later
-    -   Microsoft Edge 18
+    -   Safari V12.1, V13 or later
```
Comment (1000TurquoisePogs) [inline docs/user-guide/systemrequirements.md:132]: We need to remove the old versions, not append new ones. It's just following the explanation paragraph above.
Alternatively, remove all of the versions and just let the above paragraph explain which versions are supported.

File: docs/user-guide/systemrequirements.md [hunk 2]
```diff
@@ -69,8 +71,10 @@ For more information on the relationship between the Zowe server components and
 
 The Zowe Application Framework server provides the Zowe Desktop that contains an extensible GUI with a number of applications allowing access to z/OS functions, such as the File Editor, TN3270 emulator, JES Explorer, and more. For more information, see [Zowe Architecture](../getting-started/zowe-architecture.md#zlux).
 
-- Node.js v8.x (except v8.16.1), and v12.x.
-  **Note:** when using v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os)
+- Node.js v8.x (except v8.16.1), v12.x, or v14
+  
+  **Note:** When using Node.js v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os).
```
Comment (1000TurquoisePogs) [inline docs/user-guide/systemrequirements.md:76]: tagging will apply to every version v12 and afterward, such as v14 and whatever version comes after v14.

File: docs/user-guide/systemrequirements.md [hunk 3]
```diff
@@ -59,8 +59,10 @@ For more information on the relationship between the Zowe server components and
 
 ### Zowe Web Explorers and APIs on z/OS requirements (host)
 
-- Node.js v8.x (except v8.16.1), and v12.x.
-  **Note:** when using v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os)
+- Node.js v8.x (except v8.16.1), v12.x, or v14.
+  
+  **Note:** When using Node.js v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os).
```
Comment (1000TurquoisePogs) [inline docs/user-guide/systemrequirements.md:64]: tagging will apply to every version v12 and afterward, such as v14 and whatever version comes after v14.

File: docs/user-guide/systemrequirements.md (hunk 1 above)
Comment (nannanli) [reply docs/user-guide/systemrequirements.md:132]: Fixed. The list is kept here so users don't have to spend time checking which release is the newest version.

File: docs/user-guide/systemrequirements.md (hunk 3 above)
Comment (nannanli) [reply docs/user-guide/systemrequirements.md:64]: Fixed. Now it says `When using Node.js v12.x or later`

**Pull request merged**